modules/Bio/EnsEMBL/Production/Pipeline/Common/AnalysisSetup.pm

```diff
@@ -197,51 +197,23 @@ sub production_updates {
    ));
    $sth->fetch();
   
-   # Load species-specific analyses, overwriting the generic info
-   $sth = $dbh->prepare(
-     'SELECT ad.description, ad.display_label, aw.displayable, wd.data '.
-     'FROM analysis_description ad, species s, analysis_web_data aw '.
-     'LEFT OUTER JOIN web_data wd ON aw.web_data_id = wd.web_data_id '.
-     'WHERE ad.analysis_description_id = aw.analysis_description_id '.
-     'AND aw.species_id = s.species_id '.
-     'AND ad.logic_name = ? '.
-     'AND s.db_name = ? '.
-     'AND aw.db_type = ? '
-   );
-   $sth->execute($logic_name, $species, $db_type);
-  
-   $sth->bind_columns(\(
-     $properties{'description'},
-     $properties{'display_label'},
-     $properties{'displayable'},
-     $properties{'web_data'},
-   ));
-  
-   $sth->fetch();
    $properties{'web_data'} = eval ($properties{'web_data'}) if defined $properties{'web_data'};
   
    # Explicitly passed parameters do not get overwritten.
-   foreach my $property (keys %properties) {
-     if (! $self->param_is_defined($property)) {
-      $self->param($property, $properties{$property});
-     }      
-   }
-   if ($dbc->pass) {
-     $sth = $dbh->prepare(
-      'INSERT IGNORE INTO analysis_web_data '.
-        '(analysis_description_id, web_data_id, species_id, db_type, '.
-          'displayable, created_at, modified_at) '.
-      'SELECT '.
-        'ad.analysis_description_id, ad.default_web_data_id, s.species_id, ?, '.
-          'ad.default_displayable, NOW(), NOW() '.
-      'FROM analysis_description ad, species s '.
-      'WHERE ad.logic_name = ? AND ad.is_current = 1 AND s.db_name = ?;'
-    );
-
-    $sth->execute($db_type, $logic_name, $species);
-  } else {
-    $self->warning("Insufficient permissions to link $species and $logic_name");
-  }
+#   foreach my $property (keys %properties) {
+#     if (! $self->param_is_defined($property)) {
+#      $self->param($property, $properties{$property});
+#     }      
+#   }
```

Comment:
This section is still required; it enables the module to be passed parameters that overwrite the default ones from the production db. Maybe that's not wise, but that should be dealt with separately, and people made aware of such a change in functionality.